source/set_up_3dsecure/index.html.md.erb

```diff
@@ -2,7 +2,7 @@
 title: Set up 3D Secure
 last_reviewed_on: 2020-06-10
 review_in: 3 months
-weight: 110
+weight: 170
 ---
 
 # Set up 3D Secure (3DS)
```

Comment:
it's also 3dS2 now as well (well, for Worldpay at least). Should we say 3DS and 3DS2? Or, we could say 'Strong Customer Authentication' which is the catchall for both but is a less familiar term.

Reply:
adding to backlog